evm_loader/lib/src/types/tracer_ch_db.rs

```diff
@@ -244,22 +262,31 @@ impl ClickHouseDb {
         "#;
 
         let time_start = Instant::now();
-        let slot_opt = Self::row_opt(
-            self.client
+        let mut slot_opt = self
+            .client
+            .query(query_last_day)
+            .bind(key)
+            .bind(slot)
+            .fetch_one::<u64>()
+            .await;
+
+        if slot_opt.is_err() {
+            slot_opt = self
+                .client
                 .query(query)
                 .bind(key)
                 .bind(slot)
                 .fetch_one::<u64>()
-                .await,
-        )?;
+                .await;
+        }
 
         let execution_time = Instant::now().duration_since(time_start);
         info!(
             "get_account_rooted_slot {{ key: {key}, slot: {slot} }} sql(1) returned {slot_opt:?}, time: {} sec",
             execution_time.as_secs_f64(),
         );
 
-        Ok(slot_opt)
+        Ok(slot_opt.ok())
     }
```

Comment:
Current solution will return None on any CH error. Such behaviour is incorrect. It's expected to raise such errors and return Ok(None) if no rows were found.